services/core/shared/core/compat/sdk_v5/network.js

```diff
@@ -14,23 +14,13 @@
  *
  */
 const coreApi = require('./coreApi');
+const { resolveOperations } = require('../common/constants');
 
 const getNetworkStatus = async () => {
 	const status = await coreApi.getNetworkStatus();
 	const { offset } = status.data.genesisConfig.rewards;
 	const { distance } = status.data.genesisConfig.rewards;
-	status.data.operations = [];
-	status.data.registeredModules.forEach(liskModule => {
-		if (liskModule.transactionAssets.length) {
-			status.data.operations = status.data.operations
-				.concat(
-					liskModule.transactionAssets.map(asset => {
-						const id = String(liskModule.id).concat(':').concat(asset.id);
-						const name = liskModule.name.concat(':').concat(asset.name);
```

Comment:
Please handle the name as optional property

Reply:
Done!